tools/Datadog.Core.Tools/Datadog.Core.Tools.csproj

```diff
@@ -1,8 +1,12 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">net45;net461;netstandard2.0</TargetFrameworks>
-    <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netstandard2.0</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">net452;net461;netcoreapp2.1;netcoreapp3.0</TargetFrameworks>
```

Comment:
We generally want to use `netstandard2.0` for class libraries and `netcoreappX.Y` for executables. Any specific reason for this change?

Reply:
I couldn't reference System.Runtime.RuntimeInformation consistently with netstandard for some reason

Reply:
You need the [`System.Runtime.InteropServices.RuntimeInformation`](https://www.nuget.org/packages/System.Runtime.InteropServices.RuntimeInformation/) NuGet package.

Reply:
Can you change these from `netcoreapp#` to `netstandard2.0`?